optics_functions/rdt.py

```diff
@@ -24,21 +24,19 @@
 
 def calculate_rdts(df: TfsDataFrame, rdts: Sequence[str],
                    qx: float = None, qy: float = None, feeddown: int = 0,
-                   complex_columns: bool = True, loop_phases: bool = False,
+                   real: bool = False, loop_phases: bool = False,
```

Comment:
You undid this

Reply:
Sorry I think my git history got messy for a moment :/

Reply:
I'm done. If you want you can rebase and force push